code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -57,15 +57,16 @@
 		"Shotgun" = list(
 			/obj/item/weapon/gun/shotgun/pump/t35 = -1,
 			/obj/item/weapon/gun/shotgun/combat/standardmarine = -1,
-			/obj/item/weapon/gun/shotgun/double/marine = -1,
-			/obj/item/storage/holster/belt/ts34/full = -1,
+			/obj/item/weapon/gun/shotgun/combat/shq6 = -1,
 			/obj/item/ammo_magazine/shotgun = -1,
 			/obj/item/ammo_magazine/shotgun/buckshot = -1,
 			/obj/item/ammo_magazine/shotgun/flechette = -1,
 			/obj/item/ammo_magazine/shotgun/tracker = -1,
 			/obj/item/weapon/gun/rifle/standard_autoshotgun = -1,
 			/obj/item/ammo_magazine/rifle/tx15_flechette = -1,
 			/obj/item/ammo_magazine/rifle/tx15_slug = -1,
+			/obj/item/weapon/gun/shotgun/double/marine = -1,
+			/obj/item/storage/holster/belt/ts34/full = -1,
```

Comment:
Any particular reason why this is being moved?

Reply:
probably moved for sorting and clutter

Reply:
![image](https://github.com/tgstation/TerraGov-Marine-Corps/assets/114457130/2fcae969-ccad-4d8b-989b-696418cf9be7)
basing off of this

Reply:
Just because RU makes a change, doesn't mean it also has to be made here.
That's dumb, moving it just for the shits and giggles.